tests/test-dirs/refactor-extract-region/extraction-issue.t/run.t

```diff
@@ -0,0 +1,77 @@
+FIXME: `x` is used instead of `d_x` in the extracted function body!
+Should be: (((a + b) + ((c * x) * y)) + z) + d_x
+
+  $ $MERLIN single refactoring-extract-region -start 10:2 -end 10:31 < foo.ml
+  {
+    "class": "return",
+    "value": {
+      "start": {
+        "line": 3,
+        "col": 0
+      },
+      "end": {
+        "line": 10,
+        "col": 31
+      },
+      "content": "let fun_name1 x y a b c d_x = (((a + b) + ((c * x) * y)) + z) + x
```

Comment:
I have mixed feeling about this. First, is this a minimal example ? Do you need a, b and c to trigger the issue ? 
The more minimal the example is, when reproducing a precise error, the easier it will be to reason about for someone who would like to try and fix it later.

Also it feels like the fix would not be too hard... would it ?